package/Shaders/Effect.hlsl

```diff
@@ -507,13 +507,42 @@ cbuffer PerGeometry : register(b2)
 #		include "LightLimitFix/LightLimitFix.hlsli"
 #	endif
 
+#	define LinearSampler SampBaseSampler
+
+#	if defined(TERRA_OCC)
+#		include "TerrainOcclusion/TerrainOcclusion.hlsli"
+#	endif
+
+#	if defined(SKYLIGHTING)
```

Comment:
Why include `Skylighting`? Only `TerrainOcclusion` and `CloudShadows` are used in `ShadowSampling.hlsli`?

Reply:
I will remove it for now. It was in case we were going to add skylighting to the ambient component (probably not worth it though)